crates/cheatnet/tests/contracts/src/starknet/blocker.cairo

```diff
@@ -29,6 +32,9 @@ mod Blocker {
             self.block_number.write(block_info.block_number);
             self.block_timestamp.write(block_info.block_timestamp);
             self.sequencer_address.write(block_info.sequencer_address);
+
+            let block_hash = get_block_hash_syscall(block_info.block_number - 10).unwrap_syscall();
```

Comment:
Why are we getting the block hash of 10 blocks before and not the current one?

Reply:
https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/system-calls-cairo1/#get_block_hash

Reply:
Should we have some test verifying that we throw an error if the block number is greater than `current_block - 10` then?

Reply:
I don't think that testing syscalls behavior is in scope here

Reply:
Should it be done at all? Or is the logic just that blockifier should be handling this properly so we don't have to check?

Reply:
yes, this part is performed by blockifier